packages/ckeditor5-media-embed/src/mediaembedui.ts

```diff
@@ -95,10 +91,10 @@ export default class MediaEmbedUI extends Plugin {
 			} );
 
 			form.delegate( 'submit', 'cancel' ).to( dropdown );
-			form.urlInputView.bind( 'value' ).to( command, 'value' );
+			form.urlInputView.fieldView.bind( 'value' ).to( command, 'value' );
 
 			// Form elements should be read-only when corresponding commands are disabled.
-			form.urlInputView.bind( 'isReadOnly' ).to( command, 'isEnabled', value => !value );
+			form.urlInputView.bind( 'isEnabled' ).to( command, 'isEnabled' );
```

Comment:
I would expect that this code worked previously so please make sure that this change is needed.

Reply:
Ok I checked it and it looks like it was invalid before (probably while we were introducing labeled fields).